pkg/apis/appstacks/v1beta1/operations.go

```diff
@@ -0,0 +1,85 @@
+package v1beta1
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OperationStatusCondition ...
+// +k8s:openapi-gen=true
+type OperationStatusCondition struct {
+	LastTransitionTime *metav1.Time                 `json:"lastTransitionTime,omitempty"`
+	LastUpdateTime     metav1.Time                  `json:"lastUpdateTime,omitempty"`
+	Reason             string                       `json:"reason,omitempty"`
+	Message            string                       `json:"message,omitempty"`
+	Status             corev1.ConditionStatus       `json:"status,omitempty"`
+	Type               OperationStatusConditionType `json:"type,omitempty"`
+}
+
+// OperatedResource ...
+// +k8s:openapi-gen=true
+type OperatedResource struct {
```

Comment:
I couldn't find any usage for this struct. Why do we need it again?

Reply:
It's not used. Removed it.